doggos/fuzzy_sets/type1_fuzzy_set.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+from typing import Callable, NoReturn
+
+
+from doggos.fuzzy_sets.fuzzy_set import FuzzySet
+
+
+class Type1FuzzySet(FuzzySet):
+    """
+    Class used to represent a fuzzy set type I:
+    https://en.wikipedia.org/wiki/Fuzzy_set
+
+    Attributes
+    --------------------------------------------
+    __membership_function : Callable[[float], float]
+        membership function, determines the degree of belonging to a fuzzy set
+
+    Methods
+    --------------------------------------------
+    __call__(x: float) -> float
+        calculate the degree of belonging to a fuzzy set of an element
```

Comment:
Consider changing to: "calculate degree of membership of element to a fuzzy set"